test/1_stdlib/StringTraps.swift

```diff
@@ -42,11 +42,11 @@ StringTraps.test("endIndex/successor")
   .code {
   var s = "abc"
   var i = s.startIndex
-  ++i
-  ++i
-  ++i
+  i = i.successor()
+  i = i.successor()
+  i = i.successor()
```

Comment:
```i += 3``` does not work with an Index
```i = i.advancedBy(3)``` could be used but maybe whoever originally wrote that code wanted to use 3 statements for whatever reason.